### PR TITLE
Respect system's volume settings

### LINKED ISSUE #5: Bat alert does not respect the volume settings of the laptop and screams in full volume every time it has to give an alert.
Even if my laptop has a lower volume set, batalert screams in full volume every time.
This disturbs other people specially when I am in a meeting or working from a place where other people are also working.

### PR DESCRIPTION
**Pull request by utkarsh2102**
Eeeee, we didn't respect the system's volume settings so far. Let's get this done now! \o/

Thanks, @rajudev, for opening this issue. And thanks, @Animesh-Ghosh, for fixing stuff in espeak-ruby! \o/

Fixes: #5.

### CODE REVIEW

**Comment by utkarsh2102** [comment]
@Animesh-Ghosh, would you have a couple of minutes to review this one, please? Thanks in advance!

File: lib/batalert.rb [hunk 1]
```diff
@@ -17,19 +18,21 @@ def main
...
 
+      volume = Volumerb.value
+
       if capacity < 10 && status == 'Discharging'
         notify = Libnotify.new(summary: 'PUT ON CHARGING, YOUR BATTERY IS AT ' \
-                               "#{capacity}%.", timeout: 3, urgency: :critical)
+                                        "#{capacity}%.", timeout: 3, urgency: :critical)
         notify.update
-        speech = ESpeak::Speech.new("put on charging, your battery is at #{capacity}%.")
+        speech = ESpeak::Speech.new("put on charging, your battery is at #{capacity}%.", amplitude: volume.to_s)
```
Comment (Animesh-Ghosh) [inline lib/batalert.rb:27]: Not sure why we need to give `volume` as a String, would suggest just passing it as an Integer only.
```suggestion
        speech = ESpeak::Speech.new("put on charging, your battery is at #{capacity}%.", amplitude: volume)
```

File: lib/batalert.rb [hunk 2]
```diff
@@ -17,19 +18,21 @@ def main
...
         speech.speak
       end
 
       if capacity > 90 && status == 'Charging'
         notify = Libnotify.new(summary: 'REMOVE CHARGING, YOUR BATTERY IS AT ' \
-                               "#{capacity}%.", timeout: 3, urgency: :critical)
+                                        "#{capacity}%.", timeout: 3, urgency: :critical)
         notify.update
-        speech = ESpeak::Speech.new("remove charging, your battery is at #{capacity}%.")
+        speech = ESpeak::Speech.new("remove charging, your battery is at #{capacity}%.", amplitude: volume.to_s)
```
Comment (Animesh-Ghosh) [inline lib/batalert.rb:35]: Same here.

**Review by Animesh-Ghosh — COMMENTED** [review]
@utkarsh2102 what do you think? I think we should pass the `amplitude` as an Integer instead of a String, but in the end it's your project.
I would however still like it if we could pass an Integer.

**Comment by Animesh-Ghosh** [comment]
@utkarsh2102 any way that the PR can get merged and a new release cut?

File: lib/batalert.rb (hunk 1 above)
Comment (utkarsh2102) [reply lib/batalert.rb]: Bah! You're right. It shouldn't be a string. RuboCop autocorrected stuff for me and I got all confused later. Apologies. Fixed.

**Review by Animesh-Ghosh — APPROVED** [review]

**Pull request merged**